src/main/java/org/isf/patient/model/Patient.java

```diff
@@ -615,21 +570,13 @@ public String getSearchString() {
 		sbName.append(' ');
 		sbName.append(getCity().toLowerCase());
 		sbName.append(' ');
-		if (getAddress() != null) {
-			sbName.append(getAddress().toLowerCase()).append(' ');
-		}
-		if (getTelephone() != null) {
-			sbName.append(getTelephone()).append(' ');
-		}
-		if (getNote() != null) {
-			sbName.append(getNote().toLowerCase()).append(' ');
-		}
-		if (getTaxCode() != null) {
-			sbName.append(getTaxCode().toLowerCase()).append(' ');
-		}
+		if (getAddress() != null) sbName.append(getAddress().toLowerCase()).append(' ');
```

Comment:
why undoing the better/project standard formating?

Reply:
Yes, you are true, I tried to restore the previous version 2 times, but seems that when I save, it indents automatically the code. I'll fix later. Thanks.